functions.php

```diff
@@ -5,11 +5,13 @@
  * @package HelloElementor
  */
 
+use Elementor\Plugin;
```

Comment:
Actually here I wouldn't use a `use` statement since we're not sure when this line runs that Elementor is active. Since your usage is within an `if` block checking for `ELEMENTOR_VERSION` it would only look for `Elementor\Plugin` if it exists for sure. So I'd revert this.

Reply:
Cool.